pkg/bot/interactive/feedback.go

```diff
@@ -0,0 +1,22 @@
+package interactive
+
+import "github.com/slack-go/slack"
+
+func Feedback(botName string, user string) Message {
+	btnBuilder := buttonBuilder{botName: botName}
+	return Message{
+		Sections: []Section{
+			{
+				Base: Base{
+					Body: Body{
+						Plaintext: ":wave: Hey, what's your experience with Botkube so far?",
+					},
+				},
+				Buttons: []Button{
+					btnBuilder.ForURL("Give feedback", "https://feedback.botkube.io", slack.StylePrimary),
+				},
+			},
+		},
+		OnlyVisibleForYou: user,
```

Comment:
as the only case I see is to always send the feedback with `OnlyVisibleForYou`, it would be good to change it to `bool` and here just set it to `OnlyVisibleForYou: true`. In this way, you will be able to also get rid of the `user` in Executor func. Later we can extend it when we will have more clear use-cases for this.